hydrolib/tools/extforce_convert/converters.py

```diff
@@ -282,6 +323,10 @@ def convert(self, forcing: ExtOldForcing, time_unit: str) -> Boundary:
         forcing_model = self.convert_tim_to_bc(
             tim_model, time_unit, units=units, user_defined_names=user_defined_names
         )
+        # cmp_files = list(self.root_dir.glob(f"{poly_line.filepath.stem}*.cmp"))
+        # cmp_model = self.merge_cmp_files(cmp_files, forcing)
+        # cmp_model.quantities_names = [forcing.quantity] * len(cmp_model.quantities_names)
+        # forcing_list = CMPToForcingConverter.convert(cmp_model)
```

Comment:
This is ok for this branch: enabling this code should be done once everything is in the cli branch.